### ISSUE TITLE
LateX 4a)

### ISSUE THREAD

**Issue by abisliouk** [abisliouk/IE675b-machine-learning#16]
Implement gradient descent for MAP estimation of logistic regression with a Gaussian prior / L2 regularization with hyperparameter λ. You can reuse the methods of your solution for MLE.
